src/frontend/packages/lib_components/src/common/index.tsx

```diff
@@ -1,5 +1,6 @@
 export * from './Button';
 export * from './Button/ButtonLayout';
+export * from './ContentCard';
```

Comment:
What about created a folder `Card`, and put inside ContentCard / WhiteCard, all things look like a card ?